hs_core/templates/pages/genericresource.html

```diff
@@ -850,6 +853,28 @@ <h4 class="modal-title" id="myModalLabel">Add file to resource</h4>
                                 $res_title.text(updated_title);
                             }
                         }
+
+                        if (json_response.hasOwnProperty('metadata_status')){
+                            if(json_response.metadata_status !== $('#metadata-status').text()){
+                                $('#metadata-status').text(json_response.metadata_status);
+                                if(json_response.metadata_status == "Sufficient to make public")
+                                {
+                                    alert("Metadata Status:Sufficient to make public");
+                                    $('#metadata-status').removeClass('text-danger');
+                                    $('#metadata-status').addClass('text-success');
+                                    $('#id_sharing').find('#make-public').remove();
```

Comment:
no issue functionally, but as an optimization suggestion $('#id_sharign #make-public') would be faster, one selector query instead of two, but a search by ID inside another ID can actually be slower on some browsers, because IDs are not optimized for being within anything else, they are meant for global lookup. If you can select only on #make-public that would be best.